src/wasm_fetcher/mod.rs

```diff
@@ -35,3 +35,15 @@ pub(crate) fn parse_wasm_url(
     _ => Err(anyhow!("unknown scheme: {}", parsed_url.scheme())),
   }
 }
+
+pub(crate) async fn fetch_wasm_module(url: String) -> Result<String> {
+  let fetcher = match parse_wasm_url(
+    &url, false, //TODO: remote insecure
+    false, //TODO: remote non tls
```

Comment:
:eyes:

Reply:
Yes, this is something ugly. Once https://github.com/chimera-kube/policy-server/pull/16/ is merged this can go away, because the registry pulling option will be handled via the dockerfile.

@ereslibre : do I get it right?